src/views/signup.hbs

```diff
@@ -1,4 +1,71 @@
 <div>
-    hello from sign up
-</div>
-
+   <h1>Sign Up</h1>
+   <form id="user-info">
+      <div>
```

Comment:
instead of repeating the input with the label you can make a new partial called input field and pass your inputs as a object

Reply:
but that's okay